tests/test_apt.py

```diff
@@ -64,7 +63,7 @@ def RunAllAPTTemplates(instrument):
     yam = yaml_generator.SimInput()
     yam.input_xml = xml_file
     yam.pointing_file = pointing_file
-    yam.siaf = os.path.expandvars('$MIRAGE_DATA/nircam/reference_files/SIAF/NIRCam_SIAF_2018-01-08.csv')
+    # yam.siaf = os.path.expandvars('$MIRAGE_DATA/nircam/reference_files/SIAF/NIRCam_SIAF_2018-01-08.csv')
     yam.output_dir = out_dir
     yam.simdata_output_dir = out_dir
     yam.observation_table = observationlist_file
```

Comment:
I ran this on my branch, which has master pulled in, and got an error from the assert statement in line 81. The glob there is still looking for the old YAML filenames. If you replace it with this, it will work:
```
assert len(glob.glob(os.path.join(out_dir, 'jw{0:05d}*.yaml'.format(int(PROPOSAL_ID))))) \
        >= n_obs, 'Fewer yaml files created than observations'
```

Reply:
My intuition (which could be wrong) is that this will be fixed automatically when the `pysiaf-integration` is merged into `master`, right?

Reply:
Ohhh, I guess so! I thought it might throw a merge conflict but you're right, since that part of the file hasn't been changed. Thanks for pointing that out.

Reply:
Oh wait, actually no, because this is in `test_apt.py` and I don't think this particular assert statement was changed on `master`. If it were, I wouldn't have had the error (since I have already pulled master into my local version of this branch). Or am I still confused?

Reply:
Huh, that's odd. How was this test passing for me yesterday and today? Ah, I bet the yaml files in my tests were still specifying filenames that start with V. I'll update the code with the line you provided.

Reply:
Ok, I swapped out the old line for yours above. I had to rebase and then force push in order to get the other name convention changes, which then allowed all of the pytests to pass.

Reply:
Now Travis is failing on the check to see if the MIRAGE_DATA environment variable is set. I guess it makes sense that it fails here but not in my local check though.

Reply:
Re-applied the `@pytest.mark.xfail` decorator to the check of the MIRAGE_DATA environment variable.